src/app/shared/services/tags/template-tag.service.ts

```diff
@@ -51,4 +51,27 @@ export class TemplateTagService implements EntityTagService {
       'tags[0].key': tag.key
     });
   }
+
+  public getAgreement(template: BaseTemplateModel, lang?: string): Observable<string> {
+    let tagKey = this.keys.agreementDefault;
+
+    if (lang) {
+      const langKey = `${tagKey}.${lang}`;
+      if (template.tags.find(item => item.key === langKey)) {
+        tagKey = langKey;
+      }
+    }
+
+    return this.tagService.getTag(template, tagKey)
```

Comment:
Why do you need to make a call?
is it possible to get value from template tags?